app/models/item.rb

```diff
@@ -46,6 +47,11 @@ def mark_as_deleted
     save
   end
 
+  def update_timestamps
+    self.created_at_timestamp = Time.new.to_f * 1_000_000 unless self.created_at_timestamp
+    self.updated_at_timestamp = Time.new.to_f * 1_000_000
```

Comment:
Should we not set these to equal self.created_at.to_f and self.updated_at.to_f instead, so that they equal each other?

Reply:
From what I was able to deduce from Rails magic, `updated_at` and `created_at` are not filled in at the `before_save` stage ... and putting that in `after_save` hook would require us to double the INSERT/UPDATE operations which is not optimal.

Reply:
Or turn off timestamps for for the Item model if that's possible..maybe we even delete timestamps altogether from the Item model?

Reply:
Seems like it could be set per class: https://github.com/rails/rails/blob/291a3d2ef29a3842d1156ada7526f4ee60dd2b59/activerecord/lib/active_record/timestamp.rb#L47

Reply:
`before_update` is not triggered on save. Tried also `before_commit` but that's too late in the lifecycle. 

We discussed this internally and will have to stick with this option for now